guides/login-with-github.mdx

```diff
@@ -0,0 +1,121 @@
+## Login With GitHub
+
+If you've read the [Login With Google guide](/guides/login-with-google), you would have learned that Flipt supports many methods of authentication for users to control who has access to what in Flipt.
+
+Alongside the support for generic OIDC login, Flipt has launched support for login with GitHub in version [v1.26.0](https://github.com/flipt-io/flipt/tree/v1.26.0), through their OAuth2.0 flow. This guide will serve as a walk-through on how to set this flow up for users of Flipt in your organization.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [GitHub](https://github.com/)
+- [Ngrok](https://ngrok.com/)
+
+## Brief on OAuth2.0
+
+OAuth2.0 is an authentication standard whose goal is to allow 3rd party applications to access authorized resources from a provider. It relies on the user explicitly granting access to the 3rd party application to issue a token on behalf of the OAuth2.0 provider for authorized use.
+
+![OAuth2.0 Flow Diagram](/images/guides/login-with-github/oauth-flow-diagram.png)
+
+In comparison to OIDC, OAuth2.0 doesn't have an identity standard, so retrieving identity can vary drastically from provider to provider. Therefore, it's up to the user to consult the OAuth2.0 provider's documentation to understand if and how identity information can be retrieved.
+
+## Creating OAuth2.0 application on GitHub
+
+1. Navigate to your GitHub account, and click on `Settings` under the menu of your Profile icon
+
+2. At the bottom of the menu on the left, click on the menu option titled `Developer Settings`
+
+3. This should bring you to a page that has `OAuth Apps` as a menu option on the left, click on that and click `New OAuth App` to start creating the application
+
+4. You should be brought to a page that looks like the image below, and can start filling out the information:
+
+![OAuth2.0 App Creation](/images/guides/login-with-github/oauth-registration.png)
+
+- `Application Name`: Give your application a meaningful name
+- `Homepage URL`: Usually Flipt will be used internally by organizations, so this value depends on how you plan to expose Flipt. When in doubt you can just use the URL to your organization's home page
+- `Authorization callback URL`: This value requires a publicly accessible endpoint on the internet. Thankfully, there are tools such as [ngrok](https://ngrok.com/) that will allow public traffic to an application running locally. Since we will be running Flipt in a Docker container locally, you can fire up `ngrok` and point it to port 8080 on your local machine, with the following command:
+
+```bash
+$ ngrok http 8080
+```
+
+Copy the https URL that `ngrok` provides, and append `/auth/v1/method/github/callback` to it. This will be the value you should put for the callback URL. (e.g. https://36f1-2600-1700-1d64-daa0-444c-3f7e-ceb7-3825.ngrok-free.app/auth/v1/method/github/callback).
```

Comment:
```suggestion
Copy the https URL that `ngrok` provides, and append `/auth/v1/method/github/callback` to it. This will be the value you should put for the callback URL. (e.g. `https://36f1-2600-1700-1d64-daa0-444c-3f7e-ceb7-3825.ngrok-free.app/auth/v1/method/github/callback`).
```